scripts/initializers/order.js

```diff
@@ -19,7 +20,9 @@ initializeDropin(async () => {
   const isTokenProvided = orderRef && orderRef.length > 20;
 
   // Handle redirects for user details pages
-  if (pathname === ORDER_DETAILS_PATH || pathname === CUSTOMER_ORDER_DETAILS_PATH) {
+  if (pathname === ORDER_DETAILS_PATH
+    || pathname === CUSTOMER_ORDER_DETAILS_PATH
+    || pathname === CUSTOMER_RETURN_DETAILS_PATH) {
```

Comment:
From what I can tell, the `handleUserOrdersRedirects` function seems to not be written for the `CUSTOMER_RETURN_DETAILS_PATH` handling. Do you need to update that function?

Reply:
Nope, it's fine. Order details and return details pages will use same set of containers, and will be initialised in the same way, using order initializer. Return functionality is a part of order, and related initialisation and routing logic would be shared.